app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapControls.jsx

```diff
@@ -414,7 +416,38 @@ export default class SamplesHeatmapControls extends React.Component {
     return filterTags;
   };
 
+  renderFilterStatsInfo = () => {
+    let { filteredTaxaCount, totalTaxaCount } = this.props;
+    return (
+      <span className={cs.reportInfoMsg}>
+        Showing {filteredTaxaCount} taxa of {totalTaxaCount} preselected taxa.
+        <ColumnHeaderTooltip
+          trigger={
+            <span>
+              <InfoSIcon className={cs.infoIcon} />
+            </span>
+          }
+          content="The data included in this heatmap 
+            was preselected based on the following conditions:"
+          list={[
+            "The top 1,000 unique taxa per sample",
+            "Only taxa with at least 5 reads",
```

Comment:
I recommend putting in a code comment in here that cites the constant in the code, so that in the future if we change the constant value and search for instances, we will find this text that needs to change as well.

Of course, sharing constants with rails would be even better, though outside the scope of this PR, and not something that is easy in our web code infrastructure.

Reply:
We will review this value (and probably remove it) as part of the current Epic

Reply:
Updated the code so that both constants are passed in from the server, so we should only need to update it on the rails side now.